WordPress/wordpress-playground
# Prevent editor pairing from failing if it runs multiple times

## What is this PR doing?

It ensures editor pairing works and doesn't fail if it runs multiple times.

## What problem is it solving?

Yoast has a script that ends with the same path like the Gutenberg block editor script.
This triggered the editor pairing fix two times and caused an error because it tried to redeclare a variable.

## How is the problem addressed?

The editor pairing script check now validates if the script is part od Gutenberg/WP which should prevent it from triggering multiple times.

It also moves the declaration of `__playground_readBlobAsText` inside a function in case another script triggers it in the future.  

## Testing Instructions

- [Open this blueprint](http://localhost:5400/website-server/?plugin=gutenberg&plugin=wordpress-seo&url=/wp-admin/post.php?post=1&action=edit)
- Open a post edit page
- Confirm that both Gutenberg and the Yoast meta box load
-
